template/docker-compose.yml

```diff
@@ -1,35 +1,34 @@
-web:
-  build: .
-  env_file: .env
-  links:
-    - redis
-    - db
-  ports:
-    - "${EXTERNAL_PORT}:5000"
-  environment:
-    - DATABASE_URL=postgres://postgres@db/postgres
-    - FLASK_CONFIG=production
-    - REDISTOGO_URL=redis://redis:6379
+version: "2"
 
-db:
-  image: postgres:latest
-  volumes:
-    - data:/var/lib/postgresql
+services:
+  web:
+    build:
+      context: ./web/
+      dockerfile: Dockerfile
+    env_file: .env
+    links:
```

Comment:
I think more accurately we don't need depends_on since wait-for-postgres does what that does